docs/reuse/tutorial/setup.rst

```diff
@@ -0,0 +1,75 @@
+Install Multipass.
+
+.. seealso::
+
+   See more: `Multipass | How to install Multipass
+   <https://multipass.run/docs/install-multipass>`_
+
+Use Multipass to launch an Ubuntu VM with the name ``charm-dev``
+from the 24.04 blueprint:
+
+.. code-block:: bash
+
+    multipass launch --cpus 4 --disk 50G --memory 4G --name charm-dev 24.04
+
+Once the VM is up, open a shell into it:
+
+.. code-block:: bash
+
+    multipass shell charm-dev
+
+In order to create the rock, you'll need to install Rockcraft:
+
+.. code-block:: bash
+
+    sudo snap install rockcraft --classic
+
+``LXD`` will be required for building the rock.
+Make sure it is installed and initialised:
+
+.. code-block:: bash
+
+    sudo snap install lxd
+    lxd init --auto
+
+In order to create the charm, you'll need to install Charmcraft:
+
+.. code-block:: bash
+
+    sudo snap install charmcraft --channel latest/edge --classic
```

Comment:
Should we use `latest/edge` here or switch to stable? Rockcraft above is using stable

Reply:
Does the `stable` version of charmcraft support all the extensions? For Flask it does, but since this is under `reuse` I wanted to make sure the setup was as extension-agnostic as possible

Reply:
oh right, hmm. For some we should use edge, for others stable. Perhaps we can have edge and stable setup reuable setup instructions?

Reply:
Sure, I think that would work. Do you want me to add the second set of setup instructions to this PR?

Reply:
Let's do what we need for Flask in this PR